### PR TITLE
support elasticsearch auto re-index with zero downtime via alias switch

### PR DESCRIPTION
**Pull request by camelliazhang**
* Support elastic search auto-re-index
* Use alias switch to achieve zero down time. 
* If it is first time index created or alias does not exist yet, create new alias for the first time
* The job will be schedule running ever 12hrs, so our search data can be kept up-to-date. In the past, this was manually done every few weeks
* The index alias name can be defined in elastic search ETL job configuration file
* The index mapping data model changes for 5.5
* The index mapping file location needs to be specified in job configuration file via env variables

### CODE REVIEW

**Comment by camelliazhang** [comment]
try again

**Comment by camelliazhang** [comment]
Restart build as the previous travis build gets terminated (not sure why)

**Comment by camelliazhang** [comment]
Added changes in template of es ETL job.

File: wherehows-backend/jobs/templates/ELASTIC_SEARCH_ETL.job [hunk 1]
```diff
@@ -12,3 +12,7 @@ job.jython.load=jython/ElasticSearchIndex.py
 elasticsearch.url=your_es_url
 elasticsearch.port=your_es_port
 elasticsearch.index=your_es_index
+elasticsearch.bulk.insert.size=your_bulk_insert_chunk_size
+elasticsearch.url.request.timeout=your_es_url_request_timeout
+wh.db.max.retry.times=your_db_max_retry_times
+wh.elasticsearch.index.mapping.file = your_index_mapping_file
```
Comment (mars-lan) [inline wherehows-backend/jobs/templates/ELASTIC_SEARCH_ETL.job]: Remove the extra spaces around "="

**Comment by camelliazhang** [comment]
Removed extra spaces, thanks for pointing that out.

**Pull request merged**